src/frontend/package.json

```diff
@@ -46,14 +46,17 @@
     "webpack-cli": "3.3.11"
   },
   "dependencies": {
+    "@types/emojione": "^2.2.6",
     "draft-js": "0.11.5",
     "draft-js-anchor-plugin": "2.0.3",
     "draft-js-buttons": "2.0.2",
     "draft-js-emoji-plugin": "2.1.3",
     "draft-js-plugins-editor": "3.0.0",
     "draft-js-static-toolbar-plugin": "3.0.1",
+    "emojione": "4.5.0",
     "es6-shim": "0.35.5",
     "react": "16.13.1",
-    "react-dom": "16.13.1"
+    "react-dom": "16.13.1",
+    "regenerator-runtime": "0.13.5"
```

Comment:
Why adding this package ?

Reply:
It was to avoid the error «ReferenceError: regeneratorRuntime is not defined» caused by the async callback. I'll also remove it.